README.md

```diff
@@ -266,6 +271,7 @@ Thanks goes to these wonderful people ([emoji key][emojis]):
 | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
 | [<img src="https://avatars0.githubusercontent.com/u/8260207?v=4" width="100px;"/><br /><sub><b>Karthik Iyengar</b></sub>](https://github.com/karthikiyengar)<br />[👀](#review-karthikiyengar "Reviewed Pull Requests") [💻](https://github.com/evilsoft/crocks/commits?author=karthikiyengar "Code") [📖](https://github.com/evilsoft/crocks/commits?author=karthikiyengar "Documentation") | [<img src="https://avatars1.githubusercontent.com/u/7376957?v=4" width="100px;"/><br /><sub><b>Jon Whelan</b></sub>](https://github.com/jonwhelan)<br />[🐛](https://github.com/evilsoft/crocks/issues?q=author%3Ajonwhelan "Bug reports") [💻](https://github.com/evilsoft/crocks/commits?author=jonwhelan "Code") | [<img src="https://avatars0.githubusercontent.com/u/1466420?v=4" width="100px;"/><br /><sub><b>Benny Powers</b></sub>](http://bennypowers.com)<br />[📖](https://github.com/evilsoft/crocks/commits?author=bennypowers "Documentation") [💻](https://github.com/evilsoft/crocks/commits?author=bennypowers "Code") [👀](#review-bennypowers "Reviewed Pull Requests") | [<img src="https://avatars0.githubusercontent.com/u/1909325?v=4" width="100px;"/><br /><sub><b>Dale Francis</b></sub>](https://github.com/dalefrancis88)<br />[💻](https://github.com/evilsoft/crocks/commits?author=dalefrancis88 "Code") [👀](#review-dalefrancis88 "Reviewed Pull Requests") | [<img src="https://avatars1.githubusercontent.com/u/7926867?v=4" width="100px;"/><br /><sub><b>Premith</b></sub>](https://github.com/premithk)<br />[📖](https://github.com/evilsoft/crocks/commits?author=premithk "Documentation") | [<img src="https://avatars0.githubusercontent.com/u/6302673?v=4" width="100px;"/><br /><sub><b>Dipen Bagia</b></sub>](https://github.com/dbagia)<br />[💡](#example-dbagia "Examples") | [<img src="https://avatars3.githubusercontent.com/u/5243925?v=4" width="100px;"/><br /><sub><b>Andrew Jones</b></sub>](https://github.com/ximenean)<br />[📖](https://github.com/evilsoft/crocks/commits?author=ximenean "Documentation") |
 | [<img src="https://avatars0.githubusercontent.com/u/4858051?v=4" width="100px;"/><br /><sub><b>W. K. Seymour III</b></sub>](https://github.com/wayneseymour)<br />[📖](https://github.com/evilsoft/crocks/commits?author=wayneseymour "Documentation") |
+| [<img src="https://avatars2.githubusercontent.com/u/11702749?v=4" width="100px;"/><br /><sub><b>Eliseu Benedito Codinhoto</b></sub>](https://youtube.com/euprogramadoroficial)<br />[📖](https://github.com/evilsoft/crocks/commits?author=zeucxb "Documentation") |
```

Comment:
It looks like an extra `|` some how got added. would it be possible on your end to see if running:
`npm run contributors:generate` over the file again will help...it may have been during your rebase/merge flow.
So sorry.

Reply:
I don't know why but when I run `npm run contributors:generate` it's adding a exta `|` before wayneseymour

Reply:
hmmm. I will merge this and see what I can do on my end on another branch. If that is okay with you. Then I can report back on this PR and let you know what I find if you are curious.

How does that sound?

Reply:
Sorry about the system being such a pain.

Reply:
That's ok, I removed myself and tried to run the add again but I got the same result. Thanks again.